packages/invoices/src/OutboundShipment/DetailView/modals/hooks/usePackSizeController.tsx

```diff
@@ -13,8 +13,8 @@ const distinctSortedPackSizes = (lines: DraftOutboundLine[]): number[] =>
     new Set(
       lines
         .filter(
-          ({ onHold, availableNumberOfPacks }) =>
-            availableNumberOfPacks > 0 && !onHold
+          ({ stockLine }) =>
```

Comment:
updates from fields being on the nested `stockLine` rather than the draft line itself